### DISCUSSION TITLE
conditional patches

### DISCUSSION THREAD

**Discussion by andrewcrook** [Homebrew/discussions#5585] [Writing Formulae/Casks]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Is it possible to add a conditional to a patch based on the version of the package?
I wish to use a patch to fix an issue that stops the software from working, however, I know the next release will fix it.
But that release could be sometime away.

**Comment by SMillerDev** [comment]
> I wish to use a patch to fix an issue that stops the software from working, however, I know the next release will fix it.

If it's already in the upstream repo, just add it to a `stable do` block. There are many examples in Homebrew/core. If that doesn't work for some reason you can always add  a `ohai "patch should be removed now!" if version >= "1.2.3"`
